src/foam/strategy/StrategyReference.js

```diff
@@ -16,6 +16,7 @@ foam.CLASS({
   tableColumns: [
     'desiredModelId',
     'target',
+    'strategyPredicate',
```

Comment:
is predicate appropriate for a table column? what does it look like.

Reply:
No this was accidentally left in from some experimenting. I've removed it.